mongodb/vibe/db/mongo/connection.d

```diff
@@ -306,26 +308,50 @@ final class MongoConnection {
 	{
 		scope(failure) disconnect();
 		foreach (d; documents) if (d["_id"].isNull()) d["_id"] = Bson(BsonObjectID.generate());
-		send(OpCode.Insert, -1, cast(int)flags, collection_name, documents);
+		string collection = collection_name.canFind(".") ? collection_name.split(".")[1] : collection_name;
```

Comment:
What would be the semantic of the first part of `collection'_name` here, the database name? Since it is unused, would that result in the expected behavior if it mismatches?